packages/f-form-field/src/components/FormDropdown.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+    <div
+        :class="$style['c-formDropdown']"
+        data-test-id="formDropdown">
+        <caret-icon
+            :class="$style['c-formDropdown-icon']"
+            data-test-id="formDropdown-icon" />
+        <select
+            id="time-selection"
+            :class="$style['c-formDropdown-select']"
+            data-test-id="formDropdown-select"
+            @change="updateOption">
+            <option
+                v-for="(option, index) in dropdownOptions"
+                :key="index"
+                :data-test-id="`formDropdown-option-${index}`"
+                :value="option">
+                {{ option }}
+            </option>
+        </select>
+    </div>
+</template>
+
+<script>
+import { CaretIcon } from '@justeat/f-vue-icons';
+
+export default {
+    name: 'FormField',
+
+    components: {
+        CaretIcon
+    },
+
+    props: {
+        dropdownOptions: {
+            type: Array,
+            default: () => null
+        }
+    },
+
+    methods: {
+        updateOption (event) {
+            this.$emit('update', event.target.value);
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+.c-formDropdown {
+    .c-formDropdown-icon {
+        width: 12px;
```

Comment:
Wouldn't
```suggestion
        width: spacing(x1.5);
```
be better, or is there something I'm missing?